src/include/storage/layout.h

```diff
@@ -0,0 +1,141 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// layout.h
+//
+// Identification: src/include/storage/layout.h
+//
+// Copyright (c) 2015-18, Carnegie Mellon University Database Group
```

Comment:
If you use the `formatter.py` script, it'll add the properly formatted copyright.